shell/webpack.config.js

```diff
@@ -252,6 +252,7 @@ module.exports = () => {
       ),
       new webpack.DefinePlugin({
         'process.env.NODE_ENV': JSON.stringify(nodeEnv),
+        'process.env.UI_ENV': JSON.stringify(process.env.erda_ui_env),
```

Comment:
where is the definition of `erda_ui_env`

Reply:
it's a runtime variable, defined in application pipeline setting